src/api/deposit/DepositApi.ts

```diff
@@ -9,46 +9,36 @@ import { Receipt, TxOptionalParams } from 'types'
 import Web3 from 'web3'
 import { getProviderState, Provider, ProviderState } from '@gnosis.pm/dapp-ui'
 
+export interface ReadOnlyParams {
```

Comment:
Consider not exporting this, and naming it `UserTokenPair` then you can define and export an alias for every `<method>Params` you want

Reply:
Done.